Makefile

```diff
@@ -122,6 +123,9 @@ $(BINARY)-$(VERSION).pkg: check_fpm package_build_osx
 		--description "$(DESC)" \
 		--chdir package_build_osx
 
+docker:
+	docker build -t ${DOCKER_REPO}/${BINARY} .
```

Comment:
Also, can you leave a comment in the makefile (or here) about when this command should be run? One thing I'm not clear on is if the Dockerfile is posted to docker hub or the docker image. If the Dockerfile is posted it will not work because it has to live inside this git checkout. If the image gets built and hosted that should work fine. I guess we'll see what happens when I click merge on this PR. yay!

Reply:
i updated the braces ... make docker is just building the docker image. to make it available for everyone you ( as owner of the repo ) must publish it by 
```
docker push golift/unifi-poller:latest
```
this also might happen in the travis CI process of make **make release**
i just got linux travis builds in my repos running - i did not use osx on a build process by now